packages/e2e-tests/tests/console_dock.test.ts

```diff
@@ -1,20 +1,20 @@
-import test, { expect } from "@playwright/test";
-
-import { openDevToolsTab, startTest } from "../helpers";
+import { openDevToolsTab, openRecording, startTest } from "../helpers";
 import {
   getConsoleDockButton,
   getConsoleDockToBottomButton,
   getConsoleDockToBottomRightButton,
   getConsoleDockToLeftButton,
   verifyConsoleLayout,
 } from "../helpers/dock";
+import test, { expect } from "../testFixtureCloneRecording";
 
-const url = "doc_rr_basic.html";
+test.use({ recordingUrl: "doc_rr_basic.html" });
 
 test("console_dock: Should show the correct docking behavior for recordings with video", async ({
-  page,
+  pageWithMeta: { page, recordingId },
+  recordingUrl,
 }) => {
-  await startTest(page, url);
+  await openRecording(page, recordingUrl, recordingId);
```

Comment:
Why aren't you calling `startTest` here? I think we should keep a single "start" function.